src/data/faq.json

```diff
@@ -3188,10 +3188,15 @@
                     "anchor": "rapid_antigen_test",
                     "description": "<p>Rapid antigen tests are similar in procedure to the self-test, but are performed by trained personnel. A comparison of the two antigen test types has been provided by the RKI: <a href='https://www.rki.de/DE/Content/InfAZ/N/Neuartiges_Coronavirus/Downloads/Flyer-Antigentests.pdf?__blob=publicationFile'>Link to the comparison from the RKI</a></p><p>Not all rapid antigen tests are similarly reliable. For this reason, the Paul Ehrlich Institute has published so-called minimum criteria for rapid antigen tests: <a href='https://www.pei.de/SharedDocs/Downloads/DE/newsroom/dossiers/mindestkriterien-sars-cov-2-antigentests?__blob=publicationFile&v=9'>Link to minimum criteria</a></p><p>A positive result from a rapid antigen test is not a final diagnosis and should be confirmed by a <a href='#glossary_pcr_test'>PCR test</a>.</p>"
                 },
+                {
+                    "term": "Rapid PCR Test",
+                    "anchor": "rapid_pcr_test",
+                    "description": "Rapid PCR tests, also often referred to as \"PoC-NAT-tests\", are based on the same technology as regular <a href='#glossary_pcr_test'>PCR tests</a>, but can be evaluated quickly and on-site and are therefore particularly suitable in situations where a relatively reliable test result is required within a short time. Unlike regular PCR tests, their application is not limited to a medical laboratory. At the moment it is not possible to get a Rapid PCR Test result in the Corona-Warn-App, but the technical preparations have already been implemented in version 2.19."

```

Comment:
in English, I think the right term is "PoC-NAAT"; the double 'A' comes fro mthe long name "Nucleic **A**cid **A**mplification Test", so the trainling "test" would not be required either - but makes it more readable

Reply:
Thanks for poining that out. It has been changed in https://github.com/corona-warn-app/cwa-website/pull/2863/commits/8bfbbbe1adecd24b3108f26cffde3cea25d6459f